tests/hazmat/primitives/test_ec.py

```diff
@@ -479,7 +479,8 @@ def test_load_invalid_ec_key_from_pem(self, backend):
         # BoringSSL rejects infinity points before it ever gets to us, so it
         # uses a more generic error message.
         match = (
-            "infinity" if not backend._lib.CRYPTOGRAPHY_IS_BORINGSSL else None
+            r'infinity|invalid form' \
+                if not backend._lib.CRYPTOGRAPHY_IS_BORINGSSL else None
```

Comment:
```suggestion
            r"infinity|invalid form"
            if not backend._lib.CRYPTOGRAPHY_IS_BORINGSSL
            else None
```

Reply:
Thanks!

Now flakes seems to be fine with my changes, but fails on some other files I did not change:

- `src/cryptography/hazmat/primitives/serialization/ssh.py`
- `tests/test_interfaces.py`

Reply:
That failure is due to a mypy version issue that was fixed on `main`. You are targeting the `38.0.x` branch. You'll want to retarget this PR to `main` (it may be easiest to just close this and open a new one)